docs/examples/single_moving_target_relative_Doppler.ipynb

```diff
@@ -0,0 +1,216 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #4.    radar.vel[1] =radar_vel # moving only in the y-directior 
</pre><p>Not required anymore after update of line #3</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/rf-eng/scikit-radar/pull/2/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='6'/>